build-system/tasks/visual-diff/index.js

```diff
@@ -54,12 +56,31 @@ const ROOT_DIR = path.resolve(__dirname, '../../../');
 const WRAP_IN_IFRAME_SCRIPT = fs.readFileSync(
     path.resolve(__dirname, 'snippets/iframe-wrapper.js'), 'utf8');
 
-const preVisualDiffTasks =
-    (argv.nobuild || argv.verify_status) ? [] : ['build'];
 
 let browser_;
 let webServerProcess_;
 
+/**
+ * Runs or skips build
+ */
+function runOrSkipBuild() {
```

Comment:
I already did something similar to this in the PR that I just merged :(

Reply:
No worries :)